sklearn/datasets/_samples_generator.py

```diff
@@ -6,6 +6,8 @@
 #          G. Louppe, J. Nothman
 # License: BSD 3 clause
 
+
```

Comment:
remove this line if this is not added by `black`.

Reply:
done!